template/Routes.js

```diff
@@ -15,6 +15,7 @@ function Routes ({ match: { path } }) {
     <Layout>
       <Switch>
         <Route exact path={ path + '/' } component={ Views.%SubSection% } />
+        {/* <Route path={ path + '/:id' } component={ Views.%SubSection%Details } /> */}
```

Comment:
Do you want to align more with RESTful nomenclature and call it `%SubSection%Show`?

Reply:
What's stopping you from actually creating this view?

Reply:
Calling it `show` is a good idea. In terms of generating it, depends whether you think we use that view >50% of the time. Probably, right?